install/helm/superset/values.yaml

```diff
@@ -54,6 +54,132 @@ resources: {}
   # requests:
   #   cpu: 100m
   #   memory: 128Mi
+#Superset node configuration
+supersetNode:
+  connections: 
+    redis_host: superset-redis-headless
+    redis_port: "6379"
+    db_host: superset-postgresql
+    db_port: "5432"
+    db_user: superset
+    db_pass: superset
+    db_name: superset
+
+# -----------------------------------------------------------------------------
+# Miscellaneous parameters
+# -----------------------------------------------------------------------------
+
+init:
+  enabled: true
+  waitImage:
+    repository: busybox
+    tag: latest
+    pullPolicy: IfNotPresent
+  initscript: |-
+    superset fab create-admin \
```

Comment:
I think you got things a bit out of order here. Should be:

```
superset db upgrade
superset init
superset fab create-admin ...
```

Also, please add `|| true` to the end of the fab admin command as to make it idempotent. We want this thing to run for each upgrade of Superset so that the DB stays up to date.